packages/manager/apps/cloud/client/app/dbaas/logs/detail/streams/home/logs-streams-home.html

```diff
@@ -63,8 +63,10 @@ <h3 data-translate="logs_streams_title"></h3>
         >
             <span
                 class="oui-badge oui-badge_info"
-                data-ng-bind="ctrl.storageInfo($row)"
+                data-ng-if="$row.info.isEditable"
+                data-ng-bind="($row.info.currentStorage | bytes:2:true)"
             ></span>
+            <span data-ng-if="!$row.info.isEditable"> - </span>
```

Comment:
A stream can be shared to other accounts. In this case, guest are not allowed to edit the stream (isEditable=false), and can't get storage value since the information is only available on the Metrics token of the stream owner.